products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutVerticalWithAmPm.hpp

```diff
@@ -22,6 +22,6 @@ namespace gui
 
         auto setTime(std::time_t newTime) -> void override;
 
-        TextFixedSize *fmt{};
+        TextFixedSize *ampm{};
```

Comment:
My intention was to improve the current name and at the same time be consistent with other names like `time`, `battery` and `alarm`. If we think that these are not intuitive we should probably consider changing all of them. But I'd prefer to avoid doing this in this PR.